app/models/user.rb

```diff
@@ -7,7 +7,7 @@ class User < ApplicationRecord
          :recoverable, :rememberable, :trackable, :validatable
 
   belongs_to :draw
-  has_one :membership
+  has_one :membership, dependent: :destroy
```

Comment:
Note that we can use `dependent: :destroy` here since the group isn't destroyed when a user is hypothetically destroyed. This will ensure that all the membership / group callbacks are correctly called.